client/src/panels/panel_declarations/results/IndicatorPanel.js

```diff
@@ -102,51 +101,42 @@ const query_api = (id) => {
 export default class IndicatorPanel extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { loading: true };
+    this.state = { loading: true, show_modal: false };
   }
 
   componentDidMount() {
-    const {
-      match: {
-        params: { id },
-      },
-    } = this.props;
+    const { id } = this.props;
     query_api(id).then(() => this.setState({ loading: false }));
   }
 
   render() {
-    const {
-      match: {
-        params: { org_id, id },
-      },
-    } = this.props;
+    const { id } = this.props;
 
     const { loading } = this.state;
 
-    const subject = Subject.Dept.lookup(org_id);
     const indicator = Indicator.lookup(id);
 
     return (
-      <StandardRouteContainer
-        title={text_maker("indicator_display_title")}
-        breadcrumbs={[
-          <a href={infograph_href_template(subject, "results")}>
-            {subject.name}
-          </a>,
-          text_maker("indicator_display_title"),
-        ]}
-        description={text_maker("indicator_display_desc")}
-        route_key="_inddisp"
-      >
-        <TM k="indicator_display_title" el="h1" />
-        {loading ? (
-          <SpinnerWrapper ref="spinner" config_name={"sub_route"} />
-        ) : (
-          <Panel title={indicator.name}>
-            <IndicatorDisplay indicator={indicator} show_doc={true} />
-          </Panel>
-        )}
-      </StandardRouteContainer>
+      <Fragment>
+        <a role="button" onClick={() => this.setState({ show_modal: true })}>
+          {indicator.name}
+        </a>
```

Comment:
A11y notes on this:
  - anchors that don't have href's aren't focusable by default. Add `tabindex="0"` to let users focus on this when tab navigating
  -  the indicator name doesn't explain what interacting with the button will do. A screen reader user interacting with the button won't have the visual feedback of the modal opening when they interact with it, needs an `aria-label` attribute to tell them that interacting will move them in to a modal with further details on the indicator
  - ... bit of a stretch goal, but ideally focus would return to the anchor when the modal closes (right now it goes back to the top of the table). Could add id's to the anchors and move focus back in the `on_close_callback`

Reply:
Hmm, I guess the behaviour is potentially covered by the panel text, which says clicking on an indicator name will give more information.